src/ci/scripts/install-msys2-packages.sh

```diff
@@ -6,9 +6,6 @@ IFS=$'\n\t'
 source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 
 if isWindows; then
-    pacman -S --noconfirm --needed base-devel ca-certificates make diffutils tar \
-        binutils
-
     # Detect the native Python version installed on the agent. On GitHub
```

Comment:
Can you fold this whole file in the `install-msys2.sh` script? The only reason this was executed in a separate script was incompatibilities with git bash and msys2's bash, and that's not a problem anymore.